docs/communication/protocol/join-tournament.md

```diff
@@ -0,0 +1,92 @@
+[<- Back](../index.md)
+
+## Player Joining room
```

Comment:
Please don't add dark colored boxes, makes it difficult to read the text
![image](https://user-images.githubusercontent.com/30839188/212523850-de046942-49bf-4278-aab6-472eb45ce8a2.png)

Reply:
Solved 041a145f872847b2d7afc7c52dbced6ad8ed4577